backend/src/graphql.ts

```diff
@@ -117,12 +115,56 @@ schema.queryType({
         type: 'Post',
         args: {
           status: stringArg(),
+          search: stringArg({ required: false }),
+          language: intArg({ required: false }),
+          topic: stringArg({ required: false }),
+          skip: intArg(),
+          first: intArg(),
         },
         resolve: async (parent, args, ctx) => {
+          const filterClauses = []
+
+          if (args.language) {
+            filterClauses.push({
+              language: {
+                id: {
+                  equals: args.language,
+                },
+              },
+            })
+          }
+          if (args.topic) {
+            filterClauses.push({
+              topic: {
+                some: {
+                  name: args.topic,
+                },
+              },
+            })
+          }
+          if (args.search) {
+            filterClauses.push({
+              OR: [
+                {
+                  title: {
+                    contains: args.search,
+                  },
+                },
+                {
+                  body: {
+                    contains: args.search,
```

Comment:
As we get more posts, `LIKE` queries can be very taxing on the DB without an index, and title and body aren't indexed (I think) so we should add an issue or something to the kanban as a reminder to add indexes to these (probably after we have a path forward on the case sensitivity thing)

Reply:
Issue created! See #111